app/Events/GameIntervalEvent.php

```diff
@@ -10,7 +10,12 @@ class GameIntervalEvent extends GameEvent
     public function __construct($gameId, $users, $loot)
     {
         parent::__construct($gameId);
-        $this->users = $users;
+
+        $this->users = [];
```

Comment:
Wat is dan de reden waarom je dit doet? Je noemt het zelf al een vreemde manier, maar wat gebeurt er hierdoor dan anders?

Reply:
Ah ik ben nu ook tegen deze bug aangelopen. Hierdoor worden geen spelers meer goed ingeladen in de app